api/src/main/java/com/lifegraph/team20/controller/LifeGraphsContorller.java

```diff
@@ -21,35 +14,28 @@
 import com.lifegraph.team20.models.Content;
 import com.lifegraph.team20.models.LifeGraph;
 import com.lifegraph.team20.models.UserData;
+import com.lifegraph.team20.repository.LifeGraphsRepository;
 import com.lifegraph.team20.service.ContentService;
 
-
 @CrossOrigin(origins = "*", maxAge = 3600)
 @RestController
 @RequestMapping("/life-graphs")
 public class LifeGraphsContorller {
-@RequestMapping(value = "/search", method = RequestMethod.GET)
-	public List<UserData> userData() {
-		List<UserData> userDatas = setUserData();
-		return userDatas;
-	}
 
 	@Autowired
+	LifeGraphsRepository lifeGraphsRepository;
 
-	private JdbcTemplate jdbcTemplate;
 
-	private List<UserData> setUserData() {
-		final String sql = "select * from users inner join parent_graphs on users.id = parent_graphs.user_id";
-		return jdbcTemplate.query(sql, new RowMapper<UserData>() {
-			public UserData mapRow(ResultSet rs, int rowNum) throws SQLException{
-				return new UserData(rs.getInt("id"), rs.getString("username"), rs.getTimestamp("updated_at"));
-			}
-		});
-	}
+	@RequestMapping(method = RequestMethod.GET)
+		public ResponseEntity<List<UserData>> userData() {
+			List<UserData> userDatas = lifeGraphsRepository.setUserData();
```

Comment:
```suggestion
			List<UserData> userDatas = lifeGraphsRepository.getUserData();
```
setXXX()
というメソッド名では返却値は返されることは一般的には想定しません。
取得するのであれば、getXXX()とか、selectXXX()のようなメソッド名にしましょう。